### PR TITLE
vultr 2.20.0

### PR DESCRIPTION
**Pull request by JounQin**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/v/vultr.rb [hunk 1]
```diff
@@ -23,7 +23,8 @@ class Vultr < Formula
   depends_on "go" => :build
 
   def install
-    system "go", "build", *std_go_args(ldflags: "-s -w")
+    system "go", "build", *std_go_args(ldflags: "-s -w"), "-o", "vultr"
+    bin.install "vultr"
```
Comment (chenrui333) [inline Formula/v/vultr.rb:27]:
```suggestion
```

Comment (chenrui333) [inline Formula/v/vultr.rb:27]:
```suggestion
    system "go", "build", *std_go_args(ldflags: "-s -w")
```

**Comment by chenrui333** [comment]
@JounQin can you squash the commits into one with commit msg as `vultr 2.20.0`? Thanks!

**Comment by JounQin** [comment]
@chenrui333 Done!

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
Thanks @JounQin!

**Pull request merged**